docs/3.0rc/guides/dask-ray-task-runners.mdx

```diff
@@ -20,9 +20,6 @@ These task runners can spin up a local Dask cluster or Ray instance on the fly,
 
 Use Dask or Ray in your flows to choose the execution environment that fits your particular needs.
 
-!!! note "Remote storage"
-    To ensure that tasks executing in Dask or Ray have access to task result storage, particularly when accessing a Dask or Ray instance outside of your execution environment, configure [remote file storage](/concepts/storage/).
-
```

Comment:
hopefully removing this is the right move

Reply:
agreed